Providers/Modules/nxOMSAuditdPlugin/conf/auditd_plugin.conf

```diff
@@ -0,0 +1,29 @@
+<source>
+  @type auoms
```

Comment:
Because that's how it appears in all the examples (e.g. http://docs.fluentd.org/v0.12/articles/in_unix)